models/BaseEntity.cfc

```diff
@@ -1395,6 +1461,15 @@ component accessors="true" {
 				} );
 			} )
 			.delete();
+
+		if ( variables._hasParentEntity ) {
+			variables._wirebox
+				.getInstance( variables._parentEntity.meta.fullName )
+				.set_LoadChildren( false )
+				.findOrFail( keyValues() )
```

Comment:
It would be nice if we could skip loading the entity here and just delete it.

Reply:
I thought about that, but my concern is that an overload to `delete` within the parent entity would be bypassed, then. If the parent entity needs to do any additional work during deletion, then this catches that.